src/test/java/stepDefinitions/SEDTweaksSteps.java

```diff
@@ -12,7 +26,171 @@ public SEDTweaksSteps()
 		
 	}
 	
+	@Given("^I am on a listing's details pages$")
+	public void i_am_on_a_listing_s_details_pages() throws Throwable {
```

Comment:
Could this be parameterized? Something like "Given I am on the details page for <id> on <ENV>" where "id" & "ENV" could be used to pick and choose between different products (with ID) and different environments (with ENV)? That way we'd be able to write this function once, but use it many times?